src/xrServerEntities/object_item_abstract.h

```diff
@@ -21,6 +21,7 @@ class CObjectItemAbstract
     shared_str m_script_clsid;
 
 public:
+    virtual ~CObjectItemAbstract() = default;
     IC CObjectItemAbstract(const CLASS_ID& clsid, LPCSTR script_clsid);
```

Comment:
For the consistency, it's better to place the destructor after the constructor.
```suggestion
    IC CObjectItemAbstract(const CLASS_ID& clsid, LPCSTR script_clsid);
    virtual ~CObjectItemAbstract() = default;

```